app/root.tsx

```diff
@@ -26,5 +29,17 @@ export function Layout({ children }: { children: React.ReactNode }) {
 }
 
 export default function App() {
-  return <Outlet />;
+  const domain = import.meta.env.VITE_DOMAIN;
+  const clientID = import.meta.env.VITE_CLIENT_ID;
+  const redirect_uri = import.meta.env.VITE_REDIRECT_URI;
+  
+  return (
+    <Auth0Provider
+      domain={domain}
+      clientId={clientID}
+      authorizationParams={{redirect_uri: redirect_uri}}>
+      <Outlet />
+    </Auth0Provider>
+  );
+  //return <Outlet />;
```

Comment:
We have the git history to keep track of the old version of things- it's fine to just delete it, don't need to keep it commented out, and it avoids clutter accumulating and making things slower to work on.